config/routes.rb

```diff
@@ -73,6 +73,11 @@
         end
         resources :server_rooms, only: %i[index destroy], param: :friendly_id
         resources :server_recordings, only: %i[index]
+        resources :site_settings, only: %i[] do
+          collection do 
+            patch '/update', to: 'site_settings#update'
+          end
+        end
```

Comment:
You can just use `edit` here instead of update

Reply:
sorry not `edit` but use the `update` resource instead of manually specifying the path

Reply:
The reason I made it manually is because the `update` resource takes in a id as a param. In this case there is no id needed in the param